x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -391,6 +395,35 @@ class PackagePolicyService {
 
     return Promise.all(inputsPromises);
   }
+
+  public async runExternalCallbacks(
+    externalCallbackType: ExternalCallback[0],
+    schema: typeof NewPackagePolicySchema | typeof UpdatePackagePolicySchema,
+    newPackagePolicy: NewPackagePolicy,
+    context: RequestHandlerContext,
+    request: KibanaRequest
+  ): Promise<NewPackagePolicy> {
+    let newData = newPackagePolicy;
+
+    const externalCallbacks = appContextService.getExternalCallbacks(externalCallbackType);
+    if (externalCallbacks && externalCallbacks.size > 0) {
+      let updatedNewData: NewPackagePolicy = newData;
+
+      for (const callback of externalCallbacks) {
+        try {
+          updatedNewData = schema.validate(await callback(updatedNewData, context, request));
+        } catch (error) {
+          if (error.statusCode) {
+            error.statusCode = Number(error.statusCode);
+          }
+          throw error;
```

Comment:
if we don't do the casting to number we'll only re-throw. Which is the same as not having a `try/catch` at all.

In other words, if we drop the `Number(error.statusCode)` section, we can also drop the `try/catch`